src/applications/check-in/components/BackButton.jsx

```diff
@@ -4,20 +4,31 @@ import PropTypes from 'prop-types';
 import recordEvent from 'platform/monitoring/record-event';
 
 import { createAnalyticsSlug } from '../utils/analytics';
+import { useFormRouting } from '../hooks/useFormRouting';
 
 const BackButton = props => {
-  const { action, path } = props;
+  const { action, router } = props;
+  const { getCurrentPageFromRouter, pages } = useFormRouting(router);
+
+  const currentPage = getCurrentPageFromRouter();
+  const positionInForm = pages.indexOf(currentPage);
+  const previousPage = pages[positionInForm - 1];
```

Comment:
Could probably do a new function in the `useFormRouting` called `getPreviousPage`. I can see this being useful for the new routing stuff we that might be coming

Reply:
I feel like I was going to do that, but then after coming back to it after a day of meetings, I spaced on it. Thanks for flagging that!